### PR TITLE
Cli/fix & test templates and examples

### PR DESCRIPTION
**Pull request by dpinol**
* fix imports in cli templates which were broken after 0.14 change in entry files
* add validation of templates and examples in CI
* minor fixes
* rename .js files to .jsx which use react

### CODE REVIEW

File: .github/workflows/botonic-cli-tests.yml [hunk 1]
```diff
@@ -30,3 +30,72 @@ jobs:
...
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE: botonic-cli
+    steps:
+      - name: Checking out to current branch
+        uses: actions/checkout@v1
+      - name: Setting up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```
Comment (vanbasten17) [inline .github/workflows/botonic-cli-tests.yml:44]: should we setup it with v10?

File: packages/botonic-cli/templates/intent/tests/app.test.js [hunk 2]
```diff
@@ -13,7 +16,8 @@ const o = new BotonicOutputTester(app)
 test('TEST: (404) NOT FOUND', async () => {
   await expect(i.text('whatever')).resolves.toBe(
     o.text(
-      `Enter the generated JSON key for dialogflowV2 in plugins.js to test the bot. Try typing "hello" to start the bot.`
+      // replace with  'Try typing "hello" to start the bot.' after configuring dialogflow
```
Comment (vanbasten17) [inline packages/botonic-cli/templates/intent/tests/app.test.js:19]: did you forget to remove it?

**Review by vanbasten17 — APPROVED** [review]
Nice!!! 💯

**Review by ericmarcos — APPROVED** [review]
Well done!

File: .github/workflows/botonic-cli-tests.yml (hunk 1 above)
Comment (dpinol) [reply .github/workflows/botonic-cli-tests.yml:44]: why? 12 is the lts and our target version

File: packages/botonic-cli/templates/intent/tests/app.test.js (hunk 2 above)
Comment (dpinol) [reply packages/botonic-cli/templates/intent/tests/app.test.js:19]: no, the expected text was wrong (it was concatenating the normal message and the one which appears when dialogflow is not configure

**Pull request merged**